cgroups/systemd/apply_systemd.go

```diff
@@ -67,6 +68,17 @@ func UseSystemd() bool {
 				}
 			}
 		}
+
+		// Assume StartTransientUnit on a scope allows DefaultDependencies
```

Comment:
Thanks for tracking this down Brandon, seems like a crazy one.

On Wednesday, February 4, 2015, Brandon Philips <notifications@github.com>
wrote:

> In cgroups/systemd/apply_systemd.go
> <https://github.com/docker/libcontainer/pull/359#discussion_r24099828>:
>
> > @@ -67,6 +68,17 @@ func UseSystemd() bool {
> >  				}
> >  			}
> >  		}
> > +
> > +		// Assume StartTransientUnit on a scope allows DefaultDependencies
>
> Good point, fixed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/libcontainer/pull/359/files#r24099828>.
>